include/LightGBM/utils/common.h

```diff
@@ -31,6 +31,7 @@
 #include <unordered_map>
 #include <utility>
 #include <vector>
+#include <map>
```

Comment:
is this the only line to change it in this file? I suggest that you add this include in the cpp or header when it uses <map> instead of adding it in the common header